q2_vsearch/plugin_setup.py

```diff
@@ -17,23 +25,42 @@
 )
 
 plugin.methods.register_function(
-    function=q2_vsearch._cluster_features.cluster_features,
+    function=q2_vsearch._cluster_features.cluster_features_denovo,
     inputs={
         'table': FeatureTable[Frequency],
-        'represenative_seqs': FeatureData[Sequence]},
+        'sequences': FeatureData[Sequence]},
     parameters={
-        'id': qiime2.plugin.Float % qiime2.plugin.Range(0, 1, inclusive_start=False, inclusive_end=True)},
+        'id': qiime2.plugin.Float % qiime2.plugin.Range(
+            0, 1, inclusive_start=False, inclusive_end=True)},
     outputs=[
         ('clustered_table', FeatureTable[Frequency]),
-        ('clustered_represenative_seqs', FeatureData[Sequence])
+        ('clustered_sequences', FeatureData[Sequence]),
     ],
-    name='Clusters features at user-specified percent identity.',
-    description=('Given a feature table and the associated representative '
+    input_descriptions={
+        'table': 'The feature table to be clustered.',
+        'sequences': 'The sequences corresponding to the features in table.',
+    },
+    parameter_descriptions={
+        'id': 'The percent identity at which clustering should be performed.',
```

Comment:
I know the VSEARCH flag is `--id`, but how would you feel about `identity` here? I keep seeing this and thinking of "identifier" instead of "identity." It isn't a showstopper for me, just thought I would ask.

Reply:
I went back and forth on that... I ended up going with `id` to match the vsearch parameter name. In this case it's probably fine to not match vsearch, but in general I was thinking matching vsearch's parameter names might be a useful strategy, so users can easily refer to the vsearch documentation. It's really likely that we're going to expose other vsearch parameters as well, so it will help to easily be able to find info in their docs. 

I'm fine either way for this one - you call it @thermokarst.

Reply:
Just chatted IRL with @jairideout about this - maybe it makes sense to match the `perc-identity` parameter in [`classify-consensus-vsearch`](https://docs.qiime2.org/2017.8/plugins/available/feature-classifier/classify_consensus_vsearch/) and [`classify-consensus-blast`](https://docs.qiime2.org/2017.8/plugins/available/feature-classifier/classify_consensus_blast/), and just including a note in the `description` saying something like "This parameter maps to the `--id` flag in `vsearch`"